ui/app/src/components/PluginConfigDialog/PluginConfigDialogContainer.tsx

```diff
@@ -93,7 +108,7 @@ export function PluginConfigDialogContainer(props: PluginConfigDialogContainerPr
           onSaved();
         },
         error: ({ response }) => {
-          onSubmittingAndOrPendingChange({ isSubmitting: false });
+          onSubmittingAndOrPendingChange({ isSubmitting: false, hasPendingChanges: true });
```

Comment:
It feels like by this point it should already be true. Why do you need to set it here?